repology/parsers/parsers/sisyphus.py

```diff
@@ -44,10 +44,6 @@ def iter_parse(self, path: str, factory: PackageFactory) -> Iterable[PackageMake
                 pkg.add_links(LinkType.PACKAGE_HOMEPAGE, packagedata['homepage'])
                 pkg.add_links(LinkType.PACKAGE_RECIPE_RAW, packagedata['recipe'])
                 pkg.add_links(LinkType.PACKAGE_ISSUE_TRACKER, packagedata['bugzilla'])
-                pkg.add_links(
```

Comment:
But `PACKAGE_RECIPE` link is still required.

Reply:
The case is that if we use full link it leads to download spec file directly.
But without file name at the end (as it was stripped in previous commit) it leads to display spec file on our site.

So should I just put  the spec file download link copy to  PACKAGE_RECIPE here?

Reply:
PACKAGE_RECIPE is intended for humans and PACKAGE_RECIPE_RAW is intended for automated tools. These are not interchangeable and both are required. I suggest to either put the missing link to the dump, or switch to `packagelinks`. In fact, I'd prefer the latter as it gives more flexibility.

Reply:
I'll update our API export code to provide both links for PACKAGE_RECIPE and PACKAGE_RECIPE_RAW separately.

Reply:
JSON schema updated at API.
Parser ready.